examples/07-collaboration/04-comments/App.tsx

```diff
@@ -1,13 +1,16 @@
 "use client";
 
-import { DefaultThreadStoreAuth, YjsThreadStore } from "@blocknote/core";
+import {
+  DefaultThreadStoreAuth,
+  YjsThreadStore,
+} from "@blocknote/core/comments";
 import { BlockNoteView } from "@blocknote/mantine";
 import "@blocknote/mantine/style.css";
 import { useCreateBlockNote } from "@blocknote/react";
 import { MantineProvider, Select } from "@mantine/core";
 import { YDocProvider, useYDoc, useYjsProvider } from "@y-sweet/react";
 import { useMemo, useState } from "react";
-import { HARDCODED_USERS, MyUserType, getRandomColor } from "./userdata";
+import { HARDCODED_USERS, MyUserType, getRandomColor } from "./userdata.js";
```

Comment:
I think this breaks the next.js docs website where the example is embedded. It either breaks the build / lint / dev environment.

That's why we don't use extensions in the examples. See https://github.com/TypeCellOS/BlockNote/pull/956 for original discussion, perhaps you know a better way

Reply:
Fixed it with e5fe1f99b1fe6416019b20b2c50bbce6ecfc141a